Bottom: 9225856745e61bcd24dfd092549bb1966b8d7f49
Top:    897542f65e7684f0419c117bc1c1daed5472b65c
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-03 10:45:50 +0800

Refresh of fix

---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index 7a4c6e6..079280a 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -2907,7 +2907,7 @@ static bool vhost_notify_packed(struct vhost_dev *dev,
 	if (unlikely(!v))
 		return true;
 
-	return vhost_vring_packed_need_event(vq, vq->used_wrap_counter,
+	return vhost_vring_packed_need_event(vq, vq->last_used_wrap_counter,
 					     off_wrap, new, old);
 }
